src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -275,6 +275,8 @@ def bot_equiv : (⊥ : subgraph G).coe ≃g (⊥ : simple_graph empty) :=
   right_inv := λ v, v.elim,
   map_rel_iff' := λ a b, iff.rfl }
 
+@[simp] lemma top_adj_iff {v w : V} : (⊤ : subgraph G).adj v w ↔ G.adj v w := iff.rfl
```

Comment:
Could you move this to right after the `instance : bounded_lattice`? Ideally this would be generated by `@[simps]` on that instance, but I'm not sure if `@[simps]` would do the right thing in this case.